charts/example-configurations/minikube-values-gitlab.yaml

```diff
@@ -1,118 +1,46 @@
 # Configuration file for deploying the renku chart on minikube
```

Comment:
After your changes this file is not restricted to being used on a minikube anymore. In fact you can superimpose these values to any other deployment to make it use gitlab.com as gitlab backend. I've just tried this for a dev deployment and it works like a charm 🎉. Maybe you can adapt the name of this file, the comments and the docs accordingly.

Reply:
Cool that it worked with a dev deplyoment! 
That's a great idea, we can convert this into to a template for deployments with external gitlab instances that could go into admin docs repo.